notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb

```diff
@@ -675,17 +606,17 @@
         "id": "e8ce91147c93"
       },
       "source": [
-        "### Upload the TensorFlow Hub model to a `Vertex AI Model` resource\n",
+        "## Upload the TensorFlow Hub model to Vertex AI Model Registry\n",
         "\n",
-        "Finally, you upload the model artifacts from the TFHub model into a `Vertex AI Model` resource using the method `upload()`, with the following parameters:\n",
+        "Finally, you upload the model artifacts from the TFHub model into Vertex AI Model Registry and get a model resource object using the method `upload()`, with the following parameters:\n",
         "\n",
-        "- `display_name`: A human readable name for the `Model` resource.\n",
+        "- `display_name`: A human readable name for the model resource.\n",
         "- `artifact_uri`: The Cloud Storage location of the model package.\n",
         "- `serving_container_image_uri`: The serving container image.\n",
         "\n",
-        "Uploading a model into a Vertex AI Model resource returns a long running operation, since it may take a few moments. \n",
+        "Uploading a model into a Vertex AI Model Registry returns a long running operation, since it may take a few moments.\n",
         "\n",
-        "*Note:* When you upload the model artifacts to a `Vertex AI Model` resource, you specify the corresponding deployment container image."
+        "**Note:** When you upload the model artifacts to Vertex AI Model Registry, you specify the corresponding deployment container image."
```

Comment:
delete "you"

-  "When you upload the model artifacts to Vertex AI Model Registry, specify the corresponding deployment container image"